identity-and-access-management/secret-manager/concepts.mdx

```diff
@@ -74,6 +74,15 @@ Scaleway's Secret Manager can be managed using [APIs](https://www.scaleway.com/e
 
 <Concept>
 
+## Secret protection
+
+Secret protection is a feature that allows users with the `SecretManagerFullAccess`, `SecretManagerSecretWrite`, and `AllProductsFullAccess` [permission sets](/identity-and-access-management/iam/reference-content/permission-sets/), to enable it on your secrets, so that even though they can be read and modified, they **cannot be deleted**.
```

Comment:
Maybe start with the feature description, then premission sets? "Secret protection is a feature that prevents your secrets from being deleted. users with the blablabla permission sets can enable it...."

Reply:
I plussoie Samy.